### ISSUE TITLE
RuntimeError: `get_session` is not available when using TensorFlow 2.0.

### ISSUE THREAD

**Issue by edcaron** [seva100/optic-nerve-cnn#12]
There's a compatibility error on the U-Net, OD cup on 

> **RIM-ONE v3, cropped by OD (fold 0).ipynb (nbviewer)**

on the following code:

`if K.backend() == 'tensorflow':
    sess = K.get_session()`

it throws that error:

`'`get_session` is not available '
RuntimeError: `get_session` is not available when using TensorFlow 2.0.`

Any idea on how to fix it?

**Comment by SamuelMarks** [comment]
@seva100 Do you want more work done on this repository?

I ask because of you previous https://github.com/seva100/optic-nerve-cnn/pull/8#issuecomment-394348443

Happy to help out, but don't want to spend time contributing to a deprecated codebase…

**Comment by seva100** [comment]
@edcaron the cell which contained `if K.backend() == 'tensorflow': sess = K.get_session()` was excessive, I've deleted it in the last commit. The previous cell does absolutely the same and should work. 
Please reply if you still have issues.

**Comment by seva100** [comment]
@SamuelMarks yes, thank you for reminding me about updating this repo. I did not have time to do it before unfortunately but will try to find the time to update it with more recent code. Thanks again.
